### ISSUE TITLE
Unable to Train the Model – Gets Auto Disconnected

### ISSUE THREAD

**Issue by reneesh71-ux** [ultralytics/hub#1264]
### Search before asking

- [x] I have searched the HUB [issues](https://github.com/ultralytics/hub/issues) and found no similar bug report.

### Bug

Hi Team,

I have logged into Ultralytics using the User ID 
. Since yesterday, I have been unable to train a model. Although I have sufficient balance, the training remains on the screen at 100 epochs and, after about 5 minutes, it disconnects.

I would appreciate your assistance in resolving this issue.

Thank you,
Reneesh

<img width="1473" height="447" alt="Image" src="https://github.com/user-attachments/assets/29ca0407-2a55-49a0-b546-1b29231113ce" />

### Minimal Reproducible Example

1. Login to Ultralytics
2. Dataset Uploaded
3. Clicked Train Model > Selected the setting and Choose the server and Waited for 5 mins. Model training gets disconnected

**Comment by UltralyticsAssistant** [comment]
👋 Hello @reneesh71-ux, thank you for raising an issue about [Ultralytics HUB](https://hub.ultralytics.com) 🚀  
This is an automated response to help us triage quickly, and an Ultralytics engineer will also assist you soon 👨‍💻

Please visit our [HUB Docs](https://docs.ultralytics.com/hub/) to learn more:
- [**Quickstart**](https://docs.ultralytics.com/hub/quickstart/). Start training and deploying YOLO models with HUB in seconds.
- [**Datasets: Preparing and Uploading**](https://docs.ultralytics.com/hub/datasets/). Learn how to prepare and upload your datasets to HUB in YOLO format.
- [**Projects: Creating and Managing**](https://docs.ultralytics.com/hub/projects/). Group your models into projects for improved organization.
- [**Models: Training and Exporting**](https://docs.ultralytics.com/hub/models/). Train YOLOv5 and YOLOv8 models on your custom datasets and export them to various formats for deployment.
- [**Integrations**](https://docs.ultralytics.com/hub/integrations/). Explore different integration options for your trained models, such as TensorFlow, ONNX, OpenVINO, CoreML, and PaddlePaddle.
- [**Ultralytics HUB App**](https://docs.ultralytics.com/hub/app/). Learn about the Ultralytics App for iOS and Android, which allows you to run models directly on your mobile device.
    * [**iOS**](https://docs.ultralytics.com/hub/app/ios/). Learn about YOLO CoreML models accelerated on Apple's Neural Engine on iPhones and iPads.
    * [**Android**](https://docs.ultralytics.com/hub/app/android/). Explore TFLite acceleration on mobile devices.
- [**Inference API**](https://docs.ultralytics.com/hub/inference-api/). Understand how to use the Inference API for running your trained models in the cloud to generate predictions.

Since this appears to be a 🐛 Bug Report, thank you for sharing initial reproduction steps and a screenshot 🙏  
To help the team investigate faster, please also include a strict [minimum reproducible example](https://docs.ultralytics.com/help/minimum_reproducible_example/) with exact step-by-step actions, along with any additional screenshots of the disconnect state and relevant environment/session details from HUB.

If this is a ❓ Question, please provide as much information as possible, including dataset, model, environment details etc. so that we might provide the most helpful response.  
We try to respond to all issues as promptly as possible. Thank you for your patience!

**Comment by reneesh71-ux** [comment]
Hi Team,

Thanks for the response, I have modified the User details.

Quick Checks
1. Confirm the job is actually stopping vs. only the UI disconnecting - 0 epochs and after 5 mins it gets disconnected.
2. Verify billing balance in the same workspace - Yes, Verified - Have enough balance.
3.  Rule out browser/network session drops - Tried on different network & different machines/browsers
4.  Confirm dataset is fully processed - Yes, Generated multiple dataset.

The model URL - 
1. https://hub.ultralytics.com/models/n5o6aw23BSpbPAuabxDD
2. https://hub.ultralytics.com/models/oSoYvnNJaAQxyYllVyad
3. https://hub.ultralytics.com/models/b6ikZ9pNYQZX0xuSqM4m
4. https://hub.ultralytics.com/models/ppyEWXFs6YJq6whL8WMw
5. https://hub.ultralytics.com/models/o8Ix87BIvenPVt1hVVp5
6. https://hub.ultralytics.com/models/4zkGXYJGENYfqVkMemJV
7. https://hub.ultralytics.com/models/p4iREhkOZHJWYMzjK6hm
8. https://hub.ultralytics.com/models/bstDVutelAwaUW1MwCg2
9. https://hub.ultralytics.com/models/xnfxSXNiUoj5SSLRH6sb
10. https://hub.ultralytics.com/models/8lKiwSzKMsfqIHEeQWht

The dataset URL/ID
1. n5o6aw23BSpbPAuabxDD
2. oSoYvnNJaAQxyYllVyad
3. b6ikZ9pNYQZX0xuSqM4m
4. ppyEWXFs6YJq6whL8WMw
5. o8Ix87BIvenPVt1hVVp5
6. 4zkGXYJGENYfqVkMemJV
7. p4iREhkOZHJWYMzjK6hm
8. bstDVutelAwaUW1MwCg2
9. xnfxSXNiUoj5SSLRH6sb
10. 8lKiwSzKMsfqIHEeQWht

The region/server you selected and the approximate time it disconnected (include timezone, or UTC) - NVIDIA H200
Your browser + version and OS - Mac/Windows - Chrom & Edge
Any visible error text in the training dialog (and if possible, a screenshot of the Console subtab output) - 



I hope this detail helps. Local & Google Collab training is extremely slow. Hence would request you to fix the issue with Ultralytics. 

Also, If you are able to fix the problem from your backend. Maybe delete all Disconnected Model. So that i can start fresh. Thanks,

**Comment by pderrenger** [comment]
Thanks Reneesh and Maha — this is exactly the kind of signal we needed.

A **500 Internal Server Error** on `POST /v1/models/<MODEL_ID>/train` means the failure is happening **before** a training job can be created/queued, which explains all of the downstream symptoms you both see:

- training stays at **0 epochs**
- **no Console/System output** (nothing ever starts)
- UI flips to **“Disconnected”** (the app can’t attach to a real job/stream)
- **Delete** failing likely shares the same underlying API/service degradation

### ✅ Confirmed failing calls (captured)
- Reneesh: `POST https://api.ultralytics.com/v1/models/n5o6aw23BSpbPAuabxDD/train` → **500** with “Unhandled server error”
- Maha: `POST /v1/models/latXzaNPzT7qEUOH670f/train` → **500** at **2026-03-05 14:22:59 UTC**
- Plus your provided model/dataset IDs + timestamps (thank you)

I’ve escalated this internally as a **platform incident** with your exact **model IDs + UTC timestamps** so the team can correlate directly to server logs and the job scheduler/provisioning pipeline.

### 2 more data points that would speed up root-cause (optional, but very helpful)
If you have 60 seconds, please grab **one** failing **Delete** attempt from DevTools → Network and paste:

1) the request path (e.g. `/v1/models/<id>` or similar)  
2) the **HTTP status code** (403/404/409/500, etc.)  
3) the **UTC timestamp** of the attempt

That will tell us whether Delete is failing for the same reason as Train (same service) or a separate downstream dependency.

Also: if you *do* see any header like `x-request-id`, `x-cf-ray`, `x-amzn-trace-id`, etc. on the 500 response, that’s gold — but totally understand if it’s not present.

### About “expedite / business impact”
Understood, and you’re right to push on this — with multiple independent accounts reporting the same `/train` 500, we’re treating it as urgent. I’ll post an update in this issue as soon as we have either (a) confirmed root cause, or (b) mitigation/fix rollout status.

### Temporary workaround (only if you absolutely must make progress today)
Cloud training will keep failing as long as the API is returning 500 on `/train`. If you have any GPU locally (even a single card), the most reliable stopgap is **local training while streaming metrics to HUB** (this bypasses cloud provisioning entirely). See the [Model Training docs](https://docs.ultralytics.com/platform/train/) for the workflow, and please use the **Local Training** tab’s copy/paste command so the dataset/model identifiers are guaranteed correct.

If local/Colab is CPU-only, I agree it’s usually not a practical workaround for production timelines.

---

**Action items (if you can):**
- (Reneesh + Maha) Share **one** DevTools Network entry for the **Delete** failure (path + status + UTC time)

I’ll take it from there and keep this thread updated with the fix status.

**Comment by Maha-aouissat** [comment]
Hi @sergiuwaxmann ,
Thanks for your message.
I checked the dataset YAML file and it is correctly configured. I also verified the dataset locally with Python, and the checks completed correctly.

We have already been in contact with Paula about this issue, and based on the tests we shared, it seems to be a backend problem rather than a dataset configuration issue.

Also, the Delete action for models is not working either, which further suggests that the issue is not coming from the dataset itself.

Thanks.

**Comment by Maha-aouissat** [comment]
@sergiuwaxmann 
Hi,

I am experiencing an issue with Ultralytics Hub training that seems similar to a previous problem.

Previously, I managed to complete a training run using a YAML configuration with an empty `path` (as recommended). However, the training took an unusually long time (~19 hours), but it eventually finished successfully.

Here is the working model:
https://hub.ultralytics.com/models/ubqoclhFc0UiWwqHDpvw

Dataset used:
https://hub.ultralytics.com/datasets/jyeJ3HqUpkoD3BmcVs1h



Today, I tried to launch a new training using a similar YAML configuration (same structure, same approach with empty `path`), but the training does not start at all.

Here is the model that fails to start:
https://hub.ultralytics.com/models/ozpM9Mj7wjGQOA3yKOkS

Dataset used:
https://hub.ultralytics.com/datasets/AywzCgJXOzPXgWG2rduU


Issue details:

Training does not start
Error message: "Something went wrong. Please try again later."
Same issue occurred previously before using the empty `path` workaround
Now even with the same approach, it does not work


Additional note:
The previous successful run took significantly longer than expected (~19 hours). I'm not sure if this is related to the current issue.


Thank you for your help.

<img width="2768" height="1310" alt="Image" src="https://github.com/user-attachments/assets/73b3f720-3653-4e18-a6b2-d3f3dd747d8d" />

<img width="2758" height="1682" alt="Image" src="https://github.com/user-attachments/assets/00aeb784-8600-43da-9429-f060a8a0837f" />

**Comment by sergiuwaxmann** [comment]
@Maha-aouissat Hello! The issue you are seeing actually means that the GPU you selected is not available at that time. I agree, the message is extremely unclear... Try selecting another GPU or later.

**Comment by Maha-aouissat** [comment]
Hi, thanks for your reply
This is actually the same behavior I had previously, before using the empty path workaround....At that time switching GPUs did not resolve the issue either. (I already tried selecting different GPUs ), but the issue persists the training does not start at all)

**Comment by sergiuwaxmann** [comment]
@Maha-aouissat If the GPU availability is low, that's the error. Training doesn't even try to start, at that point, click of a buton checks GPU availability. I suggest switching to [Ultralytics Platform](https://platform.ultralytics.com) which doesn't require any special `path` in the `.yaml` file and also has more GPUs. Also, you don't even to re-upload datasets for every change as you can simply update annotations.
